internal/core/src/exec/expression/UnaryExpr.h

```diff
@@ -36,6 +39,26 @@ struct UnaryElementFunc {
             IndexInnerType;
     void
     operator()(const T* src, size_t size, IndexInnerType val, bool* res) {
+        if constexpr (op == proto::plan::OpType::Match) {
```

Comment:
this also wrapper as a function like UnaryFuncForMatch

Reply:
OK.